contrib/orc/cwrapper/format/Makefile

```diff
@@ -0,0 +1,6 @@
+All: orc_format.so
```

Comment:
this target should be "all" ?  stay consistent with top level target for "all". correct me if this can work too.